flink-staging/flink-streaming/flink-streaming-core/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamingRuntimeContext.java

```diff
@@ -91,7 +91,16 @@ public InputSplitProvider getInputSplitProvider() {
 	public Configuration getTaskStubParameters() {
 		return new TaskConfig(env.getTaskConfiguration()).getStubParameters();
 	}
-	
+
+	/**
+	 * Returns the job configuration.
+	 *
+	 * @return The job configuration.
+	 */
+	public Configuration getJobConfiguration() {
+		return new Configuration(env.getJobConfiguration());
```

Comment:
The `JobConfiguration` is a system-internal configuration. I am not sure that it is a good idea to give access to it in the `StreamingRuntimeContext`. This change will be visible for all DataStream programs.